app-emulation/cri-o/cri-o-1.9.1.ebuild

```diff
@@ -40,7 +45,13 @@ src_compile() {
 	pushd src/${EGO_PN} || die
 	GOPATH="${S}" GOBIN="${S}/bin" \
 		BASE_LDFLAGS=" -s -w -X main.gitCommit=${GIT_COMMIT} -X main.buildInfo=Gentoo" \
-		emake -j1 # -C "${S}/src/${EGO_PN}" dev
+		BUILDTAGS=""
+		if !use btrfs; then BUILDTAGS="${BUILDTAGS} exclude_graphdriver_btrfs"; fi
```

Comment:
instead of `!use` a `-n use` or `! use` will be more compatible.

Reply:
> instead of `!use` a `-n use` or `! use` will be more compatible.

Done in d175d6d →0dd1b08.